tests/providers/aws/services/rds/rds_instance_transport_encrypted/rds_instance_transport_encrypted_test.py

```diff
@@ -250,7 +250,7 @@ def test_mysql_rds_instance_no_ssl(self):
             Parameters=[
                 {
                     "ParameterName": "require_secure_transport",
-                    "ParameterValue": "0",
+                    "ParameterValue": "OFF",
```

Comment:
There are some tests failing, I don't know if they are related with this but please take a look at it. Thanks!

Reply:
I don't know why they failed, but know they work correctly.